setup.py

```diff
@@ -9,6 +9,15 @@
     long_description = f.read()
 
 
+EXTRAS_REQUIRE = {
+    'test': ['pytest', 'pytest-cov', 'pytest-mock', 'pytest-asyncio',
+             'asynctest'],
+    'fetch': ['aiohttp'],
+}
+EXTRAS_REQUIRE['all'] = [
+    vv for k, v in EXTRAS_REQUIRE.items() for vv in v]
```

Comment:
alternative: https://github.com/pvlib/pvlib-python/blob/master/setup.py#L53 I'm sure I copied that from stack overflow or another project. don't know if one is preferred.

Reply:
:man_shrugging:  i'll stick with this (and only use .values())